From 83059ee52570856ac41330ab70e5d7383a846cbb Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Wed, 6 Apr 2011 08:34:56 +0200
Subject: [PATCH] Mark Heaphone {Center|LFE} control of emu10k1 conf as
 optional

The controls referred in center_lfe definition in EMU10K1.conf should be
marked as optional since some boards don't have these.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 src/conf/cards/EMU10K1.conf |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/conf/cards/EMU10K1.conf b/src/conf/cards/EMU10K1.conf
index 7a7ebf5..956e163 100644
--- a/src/conf/cards/EMU10K1.conf
+++ b/src/conf/cards/EMU10K1.conf
@@ -108,6 +108,7 @@ EMU10K1.pcm.center_lfe.0 {
 				index 1
 				preserve true
 				# lock true
+				optional true
 				value true
 			}
 			{
@@ -115,6 +116,7 @@ EMU10K1.pcm.center_lfe.0 {
 				index 1
 				preserve true
 				# lock true
+				optional true
 				value true
 			}
 # if you have a creative's digital receiver, you can get surround/center/lfe
-- 
1.7.6

